SupremacyCore/Combat/AutomatedCombatEngine.cs

```diff
@@ -1198,7 +1494,7 @@ protected override void ResolveCombatRoundCore()
 
                     // Calculate Bonus/Malus
                     // Get Accuracy, Damage Control when fixed
-                    double sourceAccuracy = 0.70; 
+                    double sourceAccuracy = 0.70;
                     double targetDamageControl = 0.55;
 
                     // COMMAND SHIP MODFIER (retaliation loop)
```

Comment:
There is a ShipType enum. This should be used instead so that all command ships can be done with a single operation instead of coding in all the possible command ships